src/SGB_Functions.md

```diff
@@ -3,7 +3,7 @@
 ## General Description
 
 Basically, the SGB (Super Game Boy) is an adapter cartridge that allows
-to play Game Boy games on a SNES (Super Nintendo Entertainment System)
+to play Game Boy games on an SNES (Super Nintendo Entertainment System)
```

Comment:
Again, I don't think there's a clear consesus on how to pronounce SNES so I wouldn't go as far as changing these occurences

Reply:
*Nintendo Power* magazine abbreviated it as "Super NES" most often. I even vaguely remember a letter to the editor in Player's Pulse asking about this, with a reply from the editor to the effect "our style guide says 'Super NES'."

Reply:
Thanks for the feedback, good points. After searching around, it appears "a SNES" is more used than "an SNES". On the SNES Wikipedia page, "a SNES-CD" is used. I pushed a commit undoing the change. 

Leaving this conversation unresolved in case @pinobatch's point about using "Super NES" wants to be addressed further. Otherwise, a maintainer can mark this as resolved.